packages/react-storage/src/components/StorageBrowser/__tests__/createAmplifyAuthAdapter.spec.tsx

```diff
@@ -0,0 +1,58 @@
+import { act } from 'react-test-renderer';
+import { Amplify } from 'aws-amplify';
+import { Hub } from 'aws-amplify/utils';
+
+import { createAmplifyAuthAdapter } from '../adapters';
+
+describe('createAmplifyAuthAdapter', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+    jest.spyOn(Amplify, 'getConfig').mockReturnValue({
+      Storage: {
+        S3: {
+          bucket: 'XXXXXX',
+          region: 'region',
+        },
+      },
+    });
+  });
+
+  it('should create an auth adapter with the correct properties', () => {
+    const authAdapter = createAmplifyAuthAdapter();
+
+    expect(authAdapter).toHaveProperty('getLocationCredentials');
+    expect(authAdapter).toHaveProperty('listLocations');
+    expect(authAdapter).toHaveProperty('region');
+    expect(authAdapter.region).toEqual('region');
+    expect(authAdapter).toHaveProperty('registerAuthListener');
+  });
+
+  it('should throw if no bucket or region given', () => {
+    jest.spyOn(Amplify, 'getConfig').mockReturnValue({});
+
+    expect(createAmplifyAuthAdapter).toThrow(
+      'Amplify Storage configuration not found. Did you run `Amplify.configure` from your project root?'
```

Comment:
Not a big deal, but maybe we should create a constant from the error message and import here so the test doesn't need to be updated if the error message is updated